L1Trigger/TrackFindingTracklet/BuildFile.xml

```diff
@@ -0,0 +1,46 @@
+<use   name="FWCore/Framework"/>
+<use   name="FWCore/PluginManager"/>
+<use   name="FWCore/ParameterSet"/>
+<use   name="FWCore/Utilities"/>
+<use   name="CondFormats/DataRecord"/>
+<use   name="CondFormats/L1TObjects"/>
+<use   name="Geometry/Records"/>
+<use   name="Geometry/CommonDetUnit"/>
+<use   name="Geometry/CommonTopologies"/>
+<use   name="Geometry/DTGeometry"/>
+<use   name="Geometry/TrackerGeometryBuilder"/>
+<use   name="DataFormats/Common"/>
+<use   name="DataFormats/DetId"/>
+<use   name="DataFormats/GeometryVector"/>
+<use   name="DataFormats/GeometrySurface"/>
+<use   name="DataFormats/SiPixelDigi"/>
+<use   name="DataFormats/TrackReco"/>
+<use   name="DataFormats/VertexReco"/>
+<use   name="TrackingTools/TrackAssociator"/>
+<use   name="PhysicsTools/CandUtils"/>
+<use   name="FWCore/MessageLogger"/>
+<use   name="DataFormats/L1Trigger"/>
+<use   name="DataFormats/HepMCCandidate"/>
+<use   name="SimDataFormats/Vertex"/>
+<use   name="SimDataFormats/TrackingHit"/>
+<use   name="SimDataFormats/GeneratorProducts"/>
+<use   name="TrackingTools/Records"/>
+<use   name="TrackingTools/PatternTools"/>
+<use   name="TrackingTools/TrajectoryState"/>
+<use   name="TrackingTools/GeomPropagators"/>
+<use   name="TrackingTools/TransientTrack"/>
+<use   name="TrackingTools/TrajectoryParametrization"/>
+<use   name="RecoTracker/TrackProducer"/>
+<use   name="clhep"/>
+<use   name="xerces-c"/>
+<use   name="boost"/>
+<use   name="root"/>
+<use   name="rootrflx"/>
+<use   name="DataFormats/L1TrackTrigger"/>
+<!-- Add no-misleading-indentation option to avoid warnings about bug in Boost library. -->
+<flags CXXFLAGS="-g -Wno-unused-variable -Wno-misleading-indentation -Wno-maybe-uninitialized"/>
```

Comment:
See the [comment](https://github.com/cms-sw/cmssw/pull/29381/files#r403039410) by @makortel in the review of #29381: 

> Please remove `-g`. For the others special flags, could you give example of the warnings and what the bug is? Is the bug present present in Boost 1.72?

Reply:
done

Reply:
removed